### PR TITLE
Adding journalists questions dataset

### PR DESCRIPTION
**Pull request by MaramHasanain**
This is my first dataset to be added to HF.

### CODE REVIEW

File: datasets/journalists_questions/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+annotations_creators: [crowdsourced]
+language_creators: [other]
+languages: [ar]
+licenses: [unknown]
+multilinguality: [monolingual]
+size_categories: [1k<n<10K]
+source_datasets: [original]
+task_categories: [text-classification]
+task_ids: [text-classification-other-question-identification]
```
Comment (lhoestq) [inline datasets/journalists_questions/README.md:10]:
```suggestion
annotations_creators:
- crowdsourced
language_creators:
- other
languages:
- ar
licenses:
- unknown
multilinguality:
- monolingual
size_categories:
- 1k<n<10K
source_datasets:
- original
task_categories:
- text-classification
task_ids:
- text-classification-other-question-identification
```

File: datasets/journalists_questions/journalists_questions.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        )
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "tweet_id": datasets.Value("string"),
+                    "label": datasets.features.ClassLabel(names=["yes", "no"]),
```
Comment (lhoestq) [inline datasets/journalists_questions/journalists_questions.py:56]:
```suggestion
                    "label": datasets.features.ClassLabel(names=["no", "yes"]),
```
usually the positive label is at index 1

**Comment by MaramHasanain** [comment]
@lhoestq Thanks a lot for checking! I hope I addressed all your comments.

**Review by lhoestq — APPROVED** [review]
Thanks !

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**